docs/tutorials.rst

```diff
@@ -4,6 +4,12 @@
 Tutorials
 =========
 
+.. note::
+  The Simulators tutorials have moved to
+  `Qiskit Aer <https://qiskit.org/ecosystem/aer/tutorials/index.html>`_
+  and the Algorithms tutorials to
+  `Qiskit Algorithms <https://qiskit.org/ecosystem/algorithms/tutorials/index.html>`_.
```

Comment:
Does it make sense to say a little more here? Not only have they moved, to take advantage of them you also need to refactor any code you have over from `qiskit.algorithms` to `qiskit_algorithms`. There is some text in the Qiskit release notes around that - this does not link to it exactly but its in the prelude https://qiskit.org/documentation/release_notes.html#release-notes-0-25-0-prelude Also qiskit_algorithms release note says something similar https://qiskit.org/ecosystem/algorithms/release_notes.html#release-notes-0-1-0 if thats preferable.

Reply:
This seems like a warning that would better fit the documentation on the `qiskit_algorithms` side, we could add a note there warning about the change in the tutorials with respect to `qiskit`.